compiler/src/Language/Bond/Codegen/Java/Util.hs

```diff
@@ -83,17 +83,14 @@ defaultValue java Field {fieldDefault = Nothing, ..} = implicitDefault fieldType
 
 defaultValue java Field {fieldDefault = (Just def), ..} = explicitDefault def
   where
-    explicitDefault (DefaultInteger x) = Just $ intLiteral fieldType x
+    explicitDefault (DefaultInteger x) = Just $ intLiteral x
       where
-        intLiteral BT_Int8 value = [lt|#{value}|]
-        intLiteral BT_Int16 value = [lt|#{value}|]
-        intLiteral BT_Int32 value = [lt|#{value}|]
-        intLiteral BT_Int64 value = [lt|#{value}L|]
-        intLiteral BT_UInt8 value = [lt|#{value}|]
-        intLiteral BT_UInt16 value = [lt|#{value}|]
-        intLiteral BT_UInt32 value = [lt|#{value}|]
-        intLiteral BT_UInt64 value = [lt|#{value}|]
-        intLiteral _ _ = error "Java:Int:defaultValue/floatLiteral: impossible happened."
+        intMax = 2147483647
```

Comment:
Found that for Int, got spooked by "Int can represent at least 2^29", ran away. Changed.